.github/workflows/binaries.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             exit 1
           fi
 
-          stack install weeder-${{ matrix.version }} --local-bin-path .
+          stack install weeder${{ matrix.version && '-' || ''}}${{ matrix.version || '' }} --local-bin-path .
```

Comment:
Can you explain why this was necessary?

Reply:
I'm still working on it and not sure, but it seemed like Stack was unhappy with being asked to build a specific version number of a package that's pinned in extra-deps.

Reply:
Ah, I could see that. Probably when it's a git dep in particular.

One suggestion might be to move this complexity to the matrix-builder code, rather than within template interpolation. So, instead of it producing `matrix.version` have it produce `matrix.suffix` or something, which is `"-{version}"` or `""` accordingly.

Reply:
will do that refactor after Windows is done dragging ass on its build